routes/web.php

```diff
@@ -9,6 +9,6 @@
 
 Route::get('/', static fn() => view('welcome'));
 
-Route::get('/login', static fn() => view('welcome'))->name('login');
-
-
+Route::get('/cities', function () {
+    return view('cities.index');
+});
```

Comment:
It's ok for now to leave the return view in here, we don't return views in actual projects (99% of the time)